models/index.js

```diff
@@ -5,13 +5,13 @@ const foodBank = require("./foodBank");
 const foodItems = require("./foodItem");
 //users dont relate to anything
 
-// user.hasMany(foodItems, {
```

Comment:
If User to food items is a many to many relationship we will need the `through:` option in both relationship declarations along with the foreign key declared. We can create the junction table between the two or sequelize will create it f declared properly -according to the docs
_The A.belongsToMany(B, { through: 'C' }) association means that a Many-To-Many relationship exists between A and B, using table C as junction table, which will have the foreign keys (aId and bId, for example). Sequelize will automatically create this model C (unless it already exists) and define the appropriate foreign keys on it.

Note: In the examples above for belongsToMany, a string ('C') was passed to the through option. In this case, Sequelize automatically generates a model with this name. However, you can also pass a model directly, if you have already defined it._